scaladoc-js/resources/code-snippets.css

```diff
@@ -0,0 +1,232 @@
+/* Snippets */
```

Comment:
What is the purpose of these changes?

Reply:
We need to break the monolith css file into smaller ones so they could be reused at, for example, docs.scala-lang